File: lang/clojure/skeleton/test/day_02_test.clj
(ns day-02-test
  (:require
   [clojure.test :refer [deftest is testing]]
   [day-02 :refer [part1 part2]]))

(deftest day-02-test
  (testing "sample data"
    (is (= 0 (part1 "../data/02-sample.txt")))
    (is (= 0 (part2 "../data/02-sample.txt")))))

File: years/2015/elixir/test/aoc/day_02_test.exs
defmodule AoC.Day02.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day02

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day02.part_1(["../data/02.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day02.part_2(["../data/02.txt"]) == 1
    end
  end
end